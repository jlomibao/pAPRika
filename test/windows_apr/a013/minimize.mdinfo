

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2752E+02     4.8989E-01     1.5811E+00     H32       105

 BOND    =        1.2658  ANGLE   =      107.5397  DIHED      =       52.6248
 VDWAALS =      -68.6073  EEL     =     1332.3979  EGB        =     -127.4178
 1-4 VDW =        5.5130  1-4 EEL =    -2130.8473  RESTRAINT  =        0.0140
 EAMBER  =     -827.5312
 NMR restraints: Bond =    0.006   Angle =     0.006   Torsion =     0.001
===============================================================================
