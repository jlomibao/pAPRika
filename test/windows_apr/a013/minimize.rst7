defa
  125
  -0.5298964   4.9626607   1.8236295   1.0065617   4.9000259   1.8245220
  -0.8900680   4.2882005   0.5837171  -0.8951788   4.2494104   3.0397665
  -0.9097347   5.9907357   1.8385181   1.4688586   5.8937427   1.8401863
   1.3122486   4.1986913   0.5851127   1.3116232   4.1589190   3.0405173
   2.6810238   4.0824426   0.1027665   2.6748323   4.0965590   3.5533402
   0.1925298   3.8275130   3.7593281   0.1974494   3.8964311  -0.1505277
   0.1722538   3.3273685   4.8706364   0.1798809   3.4348779  -1.2783957
   2.6665165   4.0326986  -0.9921113   3.2434218   4.9644459   0.4302043
   2.6377738   4.0667073   4.6484101   3.2158207   4.9903670   3.2218653
  -4.6590515   1.9878441   1.8302943  -3.8180879   3.2758369   1.8274184
  -4.2239766   1.3014669   0.6215116  -4.2746894   1.3418908   3.0783305
  -5.7364408   2.1879422   1.8051942  -4.4350876   4.1814775   1.8036809
  -3.0205515   3.1514069   0.6148317  -3.0666993   3.1859952   3.0731976
  -2.2629328   4.2838251   0.0987313  -2.2589035   4.2994739   3.5524164
  -3.3899998   2.0819511   3.8162943  -3.2995788   2.0138583  -0.0964129
  -3.0163167   1.8405951   4.9511806  -2.8667840   1.7290562  -1.1996590
  -2.2504991   4.2300549  -0.9959817  -2.7512513   5.2101301   0.4226771
  -2.2247101   4.2648863   4.6474402  -2.7239234   5.2355263   3.2222662
  -4.0947936  -3.0746596   1.8567756  -4.8209801  -1.7185839   1.8472450
  -3.3308717  -3.0519442   0.6159016  -3.2954708  -3.0167439   3.0734727
  -4.7875364  -3.9238197   1.8784899  -5.9118746  -1.8242716   1.8619480
  -4.3696351  -1.1081774   0.6036043  -4.3386645  -1.0719307   3.0600798
  -4.9452296   0.1389082   0.1200469  -4.9604664   0.1492191   3.5560058
  -3.4815015  -1.8593122   3.7831266  -3.5441547  -1.9210845  -0.1262038
  -3.0314787  -1.6097269   4.8880067  -3.1341839  -1.7086148  -1.2543410
  -4.9051742   0.1448533  -0.9753082  -5.9863607   0.1981353   0.4576488
  -4.9306158   0.1397104   4.6516613  -5.9998364   0.1804660   3.2093496
   0.5777538  -5.0982076   1.8706157  -0.9586908  -5.0341623   1.8705973
   0.9420119  -4.3919722   0.6501973   0.9400974  -4.4146088   3.1049272
   0.9573106  -6.1264564   1.8617809  -1.4224927  -6.0272651   1.8621415
  -1.2631951  -4.3003638   0.6501041  -1.2626951  -4.3224619   3.1047847
  -2.6263140  -4.2340063   0.1395194  -2.6307564  -4.2059391   3.5904197
  -0.1464623  -4.0132403   3.8356752  -0.1448521  -3.9680111  -0.0687320
  -0.1271650  -3.5388369   4.9581306  -0.1244354  -3.4633989  -1.1780837
  -2.5913639  -4.2067384  -0.9556872  -3.1707385  -5.1243829   0.4745842
  -2.6131294  -4.1520600   4.6850560  -3.1927066  -5.0901430   3.2681895
   4.7059623  -2.1205577   1.8491473   3.8667394  -3.4096838   1.8578049
   4.3078256  -1.4727263   0.6063904   4.2808449  -1.4359195   3.0625268
   5.7838776  -2.3188862   1.8633394   4.4844803  -4.3148993   1.8782498
   3.1067020  -3.3205631   0.6175490   3.0758433  -3.2849213   3.0749609
   2.3058162  -4.4388632   0.1394333   2.3128608  -4.4138744   3.5913062
   3.3600588  -2.1480451   3.7853846   3.4148923  -2.2109594  -0.1233356
   2.9329134  -1.8629288   4.8907507   3.0228827  -1.9634841  -1.2507189
   2.2731869  -4.4070439  -0.9557222   2.7739763  -5.3721558   0.4729419
   2.2992624  -4.3589469   4.6859480   2.7984401  -5.3422514   3.2691794
   4.1452260   2.9431048   1.8309606   4.8680692   1.5850540   1.8338178
   3.3418171   2.8901615   0.6169825   3.3882523   2.9193483   3.0764419
   4.8404781   3.7901881   1.8083490   5.9590306   1.6882274   1.8094167
   4.3741857   0.9402638   0.6245531   4.4267717   0.9748970   3.0811347
   4.9901110  -0.2809811   0.1223532   5.0053043  -0.2731541   3.5595604
   3.6096488   1.7896810   3.8183445   3.5160191   1.7313718  -0.0930908
   3.2134427   1.5799613   4.9518701   3.0579139   1.4838582  -1.1951171
   4.9497116  -0.2714470  -0.9729639   6.0327571  -0.3133461   0.4589217
   4.9730579  -0.2801002   4.6551672   6.0440459  -0.3323106   3.2146644
   0.0143936  -0.0282337  -0.0438490  -0.9014954  -0.6079858  -0.2118450
   0.8741461  -0.6869735  -0.2172718   0.0481502   0.7800647  -0.7845880
   0.0438369   0.5379166   1.3868616   0.9535818   1.1354247   1.5230452
  -0.0034599  -0.5853911   2.4429404  -0.8081693   1.2141684   1.5281460
  -0.9129708  -1.1830195   2.3057132   0.8484808  -1.2616185   2.3011142
   0.0255214  -0.0210643   3.8745811  -0.8339061   0.6380230   4.0482809
  -0.0093373  -0.8307987   4.6136982   0.9418989   0.5575375   4.0438618
   0.0012286  -0.0008246  -6.0031077  -0.0010657  -0.0011009  -9.0000674
  -0.0001376   2.2000036 -11.2000228
