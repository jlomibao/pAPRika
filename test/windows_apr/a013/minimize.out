
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 04/10/2019 at 18:11:30

|   Executable path: sander
| Working directory: /home/jlomibao/pAPRika_fork/pAPRika/test/windows_apr/a013
|          Hostname: entropy.ucsd.edu

  [-O]verwriting output

File Assignments:
|  MDIN: minimize.in                                                           
| MDOUT: minimize.out                                                          
|INPCRD: cb6-but-dum.rst7                                                      
|  PARM: cb6-but-dum.prmtop                                                    
|RESTRT: minimize.rst7                                                         
|  REFC: cb6-but-dum.rst7                                                      
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: minimize.mden                                                         
| MDCRD: minimize.nc                                                           
|MDINFO: minimize.mdinfo                                                       
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

PBC MD Simulation                                                              
 &cntrl                                                                        
  imin =          1,                                                           
  ntx =           1,                                                           
  irest =         0,                                                           
  maxcyc =        5000,                                                        
  ncyc =          5000,                                                        
  dt =            0.002,                                                       
  nstlim =        5000,                                                        
  ntpr =          500,                                                         
  ntwe =          500,                                                         
  ntwr =          5000,                                                        
  ntwx =          500,                                                         
  ntxo =          1,                                                           
  ioutfm =        1,                                                           
  ntf =           2,                                                           
  ntc =           2,                                                           
  cut =           999.0,                                                       
  igb =           1,                                                           
  tempi =         298.15,                                                      
  temp0 =         298.15,                                                      
  ntt =           3,                                                           
  gamma_ln =      1.0,                                                         
  ig =            -1,                                                          
  ntp =           0,                                                           
  barostat =      2,                                                           
  ntr =           1,                                                           
  restraint_wt =  50.0,                                                        
  restraintmask = '@DUM',                                                      
  nmropt =        1,                                                           
  pencut =        -1,                                                          
  ntb =           0,                                                           
 /                                                                             
 &wt type = 'END', /                                                           
DISANG = rest.in                                                               
LISTOUT = POUT                                                                 
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
| New format PARM file being parsed.
| Version =    1.000 Date = 01/16/19 Time = 17:42:13
 NATOM  =     125 NTYPES =       7 NBONH =      46 MBONA  =      93
 NTHETH =     118 MTHETA =     158 NPHIH =     200 MPHIA  =     375
 NHPARM =       0 NPARM  =       0 NNB   =     836 NRES   =       5
 NBONA  =      93 NTHETA =     158 NPHIA =     375 NUMBND =       6
 NUMANG =      12 NPTRA  =      13 NATYP =       7 NPHB   =       0
 IFBOX  =       0 NMXRS  =     108 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are modified Bondi radii (mbondi)                                                   

|     Memory Use     Allocated
|     Real               14705
|     Hollerith            382
|     Integer            27997
|     Max Pairs              1
|     nblistReal             0
|     nblist Int             0
|       Total              225 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
Note: ig = -1. Setting random seed to   587508 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       1

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     500, ntrx    =       1, ntwr    =    5000
     iwrap   =       0, ntwx    =     500, ntwv    =       0, ntwe    =     500
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       0, igb     =       1, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.00000, gbgamma =   0.00000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  50.00000

Energy minimization:
     maxcyc  =    5000, ncyc    =    5000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

NMR refinement options:
     iscale  =       0, noeskp  =       1, ipnlty  =       1, mxsub   =       1
     scalm   = 100.00000, pencut  =  -1.00000, tausw   =   0.10000
|  INFO: Old style inpcrd file read


    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

  defa
     Mask @DUM; matches     3 atoms

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

default_name                                                                    
 begin time read from input coords =     0.000 ps



           Begin reading energy term weight changes/NMR restraints
 WEIGHT CHANGES:
                         ** No weight changes given **

 RESTRAINTS:
 Requested file redirections:
  DISANG    = rest.in
  LISTOUT   = POUT
 Restraints will be read from file: rest.in
Here are comments from the DISANG input file:

                       Number of restraints read =     9

                  Done reading weight changes/NMR restraints


 Number of triangulated 3-point waters found:        0

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -8.1356E+02     1.0057E+01     2.4770E+01     N12        57

 BOND    =       11.0328  ANGLE   =      109.3438  DIHED      =       52.9822
 VDWAALS =      -67.7110  EEL     =     1326.2731  EGB        =     -123.8292
 1-4 VDW =        5.8710  1-4 EEL =    -2127.5276  RESTRAINT  =        0.0000
 EAMBER  =     -813.5649
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -8.2743E+02     4.9743E-01     1.6481E+00     H7        120

 BOND    =        1.2544  ANGLE   =      107.5594  DIHED      =       52.5889
 VDWAALS =      -68.4517  EEL     =     1332.0624  EGB        =     -127.3784
 1-4 VDW =        5.4842  1-4 EEL =    -2130.5661  RESTRAINT  =        0.0119
 EAMBER  =     -827.4468
 NMR restraints: Bond =    0.006   Angle =     0.005   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -8.2747E+02     4.9340E-01     1.5808E+00     H32       105

 BOND    =        1.2649  ANGLE   =      107.5459  DIHED      =       52.6168
 VDWAALS =      -68.5421  EEL     =     1332.2951  EGB        =     -127.4154
 1-4 VDW =        5.5037  1-4 EEL =    -2130.7502  RESTRAINT  =        0.0121
 EAMBER  =     -827.4813
 NMR restraints: Bond =    0.006   Angle =     0.005   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500      -8.2749E+02     4.9680E-01     1.5809E+00     H32       105

 BOND    =        1.2765  ANGLE   =      107.5415  DIHED      =       52.6209
 VDWAALS =      -68.5731  EEL     =     1332.3712  EGB        =     -127.4406
 1-4 VDW =        5.5069  1-4 EEL =    -2130.8059  RESTRAINT  =        0.0125
 EAMBER  =     -827.5027
 NMR restraints: Bond =    0.007   Angle =     0.005   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -8.2750E+02     4.9080E-01     1.5808E+00     H32       105

 BOND    =        1.2645  ANGLE   =      107.5416  DIHED      =       52.6216
 VDWAALS =      -68.5876  EEL     =     1332.3721  EGB        =     -127.4172
 1-4 VDW =        5.5109  1-4 EEL =    -2130.8213  RESTRAINT  =        0.0130
 EAMBER  =     -827.5154
 NMR restraints: Bond =    0.007   Angle =     0.005   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2500      -8.2751E+02     4.9039E-01     1.5809E+00     H32       105

 BOND    =        1.2660  ANGLE   =      107.5406  DIHED      =       52.6226
 VDWAALS =      -68.5971  EEL     =     1332.3870  EGB        =     -127.4197
 1-4 VDW =        5.5118  1-4 EEL =    -2130.8346  RESTRAINT  =        0.0134
 EAMBER  =     -827.5233
 NMR restraints: Bond =    0.006   Angle =     0.005   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3000      -8.2751E+02     5.1457E-01     1.5811E+00     H32       105

 BOND    =        1.2943  ANGLE   =      107.5370  DIHED      =       52.6244
 VDWAALS =      -68.6042  EEL     =     1332.4437  EGB        =     -127.4738
 1-4 VDW =        5.5076  1-4 EEL =    -2130.8564  RESTRAINT  =        0.0138
 EAMBER  =     -827.5274
 NMR restraints: Bond =    0.006   Angle =     0.006   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3500      -8.2752E+02     4.8989E-01     1.5811E+00     H32       105

 BOND    =        1.2658  ANGLE   =      107.5397  DIHED      =       52.6248
 VDWAALS =      -68.6073  EEL     =     1332.3979  EGB        =     -127.4178
 1-4 VDW =        5.5130  1-4 EEL =    -2130.8472  RESTRAINT  =        0.0140
 EAMBER  =     -827.5312
 NMR restraints: Bond =    0.006   Angle =     0.006   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4000      -8.2752E+02     4.8989E-01     1.5811E+00     H32       105

 BOND    =        1.2658  ANGLE   =      107.5397  DIHED      =       52.6248
 VDWAALS =      -68.6073  EEL     =     1332.3979  EGB        =     -127.4178
 1-4 VDW =        5.5130  1-4 EEL =    -2130.8472  RESTRAINT  =        0.0140
 EAMBER  =     -827.5312
 NMR restraints: Bond =    0.006   Angle =     0.006   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4500      -8.2752E+02     4.8989E-01     1.5811E+00     H32       105

 BOND    =        1.2658  ANGLE   =      107.5397  DIHED      =       52.6248
 VDWAALS =      -68.6073  EEL     =     1332.3981  EGB        =     -127.4178
 1-4 VDW =        5.5130  1-4 EEL =    -2130.8474  RESTRAINT  =        0.0140
 EAMBER  =     -827.5312
 NMR restraints: Bond =    0.006   Angle =     0.006   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2752E+02     4.8989E-01     1.5811E+00     H32       105

 BOND    =        1.2658  ANGLE   =      107.5397  DIHED      =       52.6248
 VDWAALS =      -68.6073  EEL     =     1332.3979  EGB        =     -127.4178
 1-4 VDW =        5.5130  1-4 EEL =    -2130.8473  RESTRAINT  =        0.0140
 EAMBER  =     -827.5312
 NMR restraints: Bond =    0.006   Angle =     0.006   Torsion =     0.001
===============================================================================


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2752E+02     4.8989E-01     1.5811E+00     H32       105

 BOND    =        1.2658  ANGLE   =      107.5397  DIHED      =       52.6248
 VDWAALS =      -68.6073  EEL     =     1332.3979  EGB        =     -127.4178
 1-4 VDW =        5.5130  1-4 EEL =    -2130.8473  RESTRAINT  =        0.0140
 EAMBER  =     -827.5312
 NMR restraints: Bond =    0.006   Angle =     0.006   Torsion =     0.001
===============================================================================
 ------------------------------------------------------------------------------


 Final Restraint Analysis for coords: minimize.rst7                           


 Restraints, deviations, and energy contributions:    pencut =   -1.00

 ------------------------------------------------------------------------------
     First atom        Last atom    curr. value target deviation  penalty
 ------------------------------------------------------------------------------
  DUM  DM1    3 --  C7   CB6    1:    9.308    9.302    0.006    0.000 d    0: 0
  DUM  DM2    4 --  C7   CB6    1:  147.243  147.398    0.154    0.001 a
  DUM  DM2    4 --  DUM  DM1    3:   49.492   49.483    0.009    0.000 t
  DUM  DM1    3 --  C31  CB6    1:   62.004   62.003    0.001    0.000 a
  DUM  DM1    3 --  C7   CB6    1: -146.032 -146.178    0.147    0.001 t
  C7   CB6    1 --  C31  CB6    1:  -72.691  -72.548    0.143    0.001 t
  DUM  DM1    3 --  C    BUT    2:    5.959    6.000    0.041    0.006 d    0: 0
  DUM  DM2    4 --  C    BUT    2:  179.719  180.000    0.281    0.002 a
  DUM  DM1    3 --  C3   BUT    2:  179.630  180.000    0.370    0.003 a
                                       Total distance penalty:      0.006
                                       Total angle    penalty:      0.006
                                       Total torsion  penalty:      0.001
|                               RMS deviation from ideal bonds :      0.0096
|                               RMS deviation from ideal angles:      4.465
 ------------------------------------------------------------------------------

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|                Calc gb radii              2.33 (22.62% of Gen B)
|                Calc gb diag               4.08 (39.52% of Gen B)
|                Calc gb off-diag           3.90 (37.83% of Gen B)
|                Other                      0.00 ( 0.02% of Gen B)
|             Gen Born time             10.31 (100.0% of Nonbo)
|          Nonbond force             10.31 (89.78% of Force)
|          Bond/Angle/Dihedral        1.14 ( 9.90% of Force)
|          Other                      0.04 ( 0.32% of Force)
|       Force time                11.49 (100.0% of Runmd)
|    Runmd Time                11.49 (99.75% of Total)
|    Other                      0.03 ( 0.25% of Total)
| Total time                11.52 (100.0% of ALL  )

| Number of list builds   :          0

| Highest rstack allocated:          0
| Highest istack allocated:          0
|           Job began  at 18:11:30.583  on 04/10/2019
|           Setup done at 18:11:30.589  on 04/10/2019
|           Run   done at 18:11:42.101  on 04/10/2019
|     wallclock() was called   90028 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
